documentation/modules/compatibility-guidelines.adoc

```diff
@@ -28,6 +28,11 @@ Therefore, it is recommended to upgrade {rhv-short} to the supported version abo
 However, migrations from {rhv-short} 4.3.11 were tested with {project-short} 2.3, and may work in practice in many environments using {project-short} {project-version}. In this case, we advise upgrading {rhv-full} Manager (RHVM) to the previously mentioned supported version before the migration to {virt}.
 ====
 
+[NOTE]
+====
+Deployment of MTV 2.5.3 and later is enabled on OpenShift Kubernetes Engine (OKE). For more information, see link:https://docs.openshift.com/container-platform/4.14/welcome/oke_about.html[About OpenShift Kubernetes Engine].
```

Comment:
Already in [MTV 2.5 docs](https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.5/html/installing_and_using_the_migration_toolkit_for_virtualization/prerequisites#compatibility-guidelines_mtv)

![image](https://github.com/kubev2v/forklift-documentation/assets/106804821/9df2de79-e340-4cf3-9315-684591d6a113)

Am I ok not to sync this?

Reply:
@anarnold97 I was thinking in terms of main -- you're right, this note should already be there in the 2.5 branch. The project-z-version attribute will need to change to 2.5.7, though. Let me fix the PR so it only includes that.

Reply:
@anarnold97 I am confused. I need to create a PR in order to change the value of {project-z-version}. Since I am working against main, not 2.5, for safety's sake, if I know remove this addition, will I cause a merge conflict against the current 2.5, which will cause you extra work? Let me know which will cause you less work -- my removing the note from this PR or leaving it. Thanks.

Reply:
How are you creating merge conflicts when we are NOT merging your PR.

You are only changing the attributes.

```
```
11 :ocp-version: 4.12
....
19 :project-version: 2.5
20 :project-z-version: 2.5.7
```

That is all I need.

You have confused yourself and the situation by not checking the 2.5 documentation, and adding an extra commit for the admonition.

Reply:
@anarnold97 Actually, all that needs to change is the value of {project-z-version} -> 2.5.7  -- the value of {ocp-version} for 2.5 is 4.12 already.

@ahadas Please confirm that this is only change needed for attributes.

Reply:
ah I missed this thread - yeah, this is the only change needed for attributes